docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/Cache.cs

```diff
@@ -8,56 +8,78 @@ public static class Cache
     {
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-            // as a catalog of available operations and as the source of randomness.
+            // Create a new context for ML.NET operations. It can be used for except
+            // ion tracking and logging, as a catalog of available operations and as
+            // the source of randomness.
             var mlContext = new MLContext();
 
             var data = DatasetUtils.LoadHousingRegressionDataset(mlContext);
 
             // Time how long it takes to page through the records if we don't cache.
-            (int lines, double columnAverage, double elapsedSeconds) = TimeToScanIDataView(mlContext, data);
-            Console.WriteLine($"Lines={lines}, averageOfColumn0={columnAverage:0.00} and took {elapsedSeconds} seconds.");
+            (int lines, double columnAverage, double elapsedSeconds) =
+                TimeToScanIDataView(mlContext, data);
+
+            Console.WriteLine($"Lines={lines}," +
+                $"averageOfColumn0={columnAverage:0.00}and took {elapsedSeconds}" +
```

Comment:
A space went missing:
```suggestion
                $"averageOfColumn0={columnAverage:0.00} and took {elapsedSeconds}" +
```